core/src/classes/tasks/clsTask.ts

```diff
@@ -7,6 +7,7 @@ import { CLS } from "../../modules/cls";
 export abstract class CLSTask extends Task {
   constructor() {
     super();
+    this.plugins = ["QueueLock"];
```

Comment:
This feels right. I'm not aware of any thing should exist twice with the same args. Any one them you are worried about?

Reply:
I don't think so!